Makefile

```
@@ -9,6 +9,8 @@ GOLANGCI_VERSION := v1.23.6
 HAS_GOLANGCI := $(shell which golangci-lint)
 GOPATH := $(shell go env GOPATH)
 
+DOCKER_CLI_EXPERIMENTAL ?= enabled
```

```suggestion
export DOCKER_CLI_EXPERIMENTAL=enabled
```
Env variable needs to be passed to child process.